kbuild/kbuild.sh

```diff
@@ -317,9 +317,9 @@ setup_cross_vars()
 			# for earlier gcc version, will failed to find arceb-elf for
 			# big endian arceb-elf tool chain
 			if is_config_enabled CONFIG_CPU_BIG_ENDIAN; then
-				crosstool=arceb-elf
+				crosstool=arceb-linux
```

Comment:
Thanks a lot, the entry of gcc-14 is different than old ones. Looks can do mapping based on the compiler as well.
```
./gcc-9.3.0-nolibc/arceb-elf
./gcc-9.3.0-nolibc/arc-elf
./gcc-10.5.0-nolibc/arc-linux
./gcc-11.5.0-nolibc/arc-linux
./gcc-13.3.0-nolibc/arc-linux
./gcc-14.2.0-nolibc/arc-linux
```
Is it possible to check whether the compiler version is >= 10, then to use arc-linux? BTW: there's no arceb-linux, in higher version of gcc, it only has arc-linux